src/devhub/devhub.js

```diff
@@ -36,11 +36,11 @@ function main_release_rotation() {
       "cb22",
       "chaitanyabhandari",
       "fabioarnold",
-      "GeorgKreuzmayr",
       "lewisdaly",
       "matklad",
       "sentientwaffle",
       "toziegler",
+      "GeorgKreuzmayr",
```

Comment:
This won't work, as we sort the candidates (so, it is correct that the order is alphabetical). However, the `shift` variable controls how you "rotate" the ring of release managers, which allows you to make a ring bigger, while maintaining the relevant position. 

You can use number theory to figure the correct value of `shift`, but I usually figure it out experimentally! (devhub is easy to run locally: `python3 -m http.server -d src/devhub/`)

Reply:
(note that even if we didn't sort, the trick of adding yourself _after_ the current release manager also wouldn't work, because we change the modulo! number theory is quite unforgiving for this one (it did took me a couple of tries to get the `shift` thing right))

Reply:
Oh, I realized that our release rotation has a flaw that there's a discontinuity at the new year! We've never noticed it, because we don't do releases around that time, but, ideally, we shouldn't use year's week, and instead use absolute week (number of days since jan 1 1970 divided by seven). Out of scope for this PR!

Reply:
Sorry, I am blind, I didn't notice that you've also updated the shift variable, my bad!